addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/PolicyTag.java

```diff
@@ -29,25 +29,37 @@
  * @since 1.29.0
  */
 public enum PolicyTag {
-    DEV_CICD,
-    DEV_STD,
-    DEV_FULL,
-    QA_STD,
-    QA_FULL,
-    SEQUENCE,
-    API;
+    DEV_CICD("Dev CICD.policy", "Developer CI/CD"),
+    DEV_STD("Dev Standard.policy", "Developer Standad"),
+    DEV_FULL("Dev Full.policy", "Developer Full"),
+    QA_STD("QA Std.policy", "QA Standard"),
```

Comment:
Before this is in wide use should we standardize on "Standard" vs "Std" ?

Reply:
Both the file and policy names already use Standard.

Reply:
So I'll switch the QA one in this PR?

Reply:
Ok apparently I made that up in my head 😞